server/graphql/general/src/queries/store.rs

```diff
@@ -133,6 +134,7 @@ impl StoreFilterInput {
             name,
             name_code,
             site_id,
+            ..
```

Comment:
Think this is a left over(?) and should be removed all filter options should be used...